docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -78,14 +78,13 @@ Not applicable as this is the initial release of GDC 2.0.
         * Keyboard focus is not returned to the triggering element when modals are closed.
         * Assistive technologies may not behave correctly with some controls due to incorrect, missing, or redundant labels, attributes, or roles.
 * Cohorts:
-    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](/Data_Portal/Users_Guide/quick_start.md#main-toolbar) locally.
+    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](/Data_Portal/Users_Guide/getting_started.md#main-toolbar) locally.
     * Cohorts created based on CNV losses or gains may not have the correct composition when filtered by additional mutated genes. As a workaround, first filter by the mutated genes before creating cohorts based on CNV losses and gains.
     * Cohorts filtered by mutated genes and SSMs not in those genes may unexpectedly result in 0 cases. <!--SV-2331/PEAR-1616-->
     * When saving a cohort, the confirmation notification may be automatically dismissed before the saving dialog has closed. <!--SV-2366-->
     * Using "Save As" to replace a cohort with itself will result in an error notification despite the replacement being successful. <!--SV-2363--> 
-* Cohort Builder:
-    * Filters related to numeric values may display a smaller number than what the user entered within the Cohort Builder. This is a visual issue and does not affect the filters applied to the cohort. <!--SV-2383-->
-    * When a selection has been made in an Available Data filter within the Cohort Builder, other Available Data filters may not display the expected values. <!--PEAR-1449-->
+    * Cohorts containing FM-AD cases may not update correctly when users with dbGaP access to FM-AD (phs001179) log in or out. As a workaround, logging in before creating cohorts with FM-AD cases is recommended. <!--SV-2389-->
+    * Cohorts with mutated gene or SSM filters may not display data in Mutation Frequency and the summary charts if filters are removed from the cohort such that they temporarily result in 0 cases while the filters are being removed. As a workaround, remove the gene and SSM filters, then add them back to the cohorts. <!--SV-2414-->
```

Comment:
I think I need help with SV-2414 phrasing... 🙏 @wwysoc2

Reply:
"If gene/mutation filters are removed from a cohort and temporarily results in 0 cases, cohorts may not display data in Mutation Frequency and the summary charts. As a workaround, remove the gene and mutation filters, then add them back."

Does this look okay?